.env

```diff
@@ -4,5 +4,6 @@ export SECRET_KEY_BASE=aBi07mDCHwwdqkunV/mx7LlDilXqy7GeZlrXMOm1Qxtdx9e6KfZ2BsF7O
 export TARGET_EMAIL=jackacarlisle@gmail.com
 export SES_SERVER=email-smtp.eu-west-1.amazonaws.com
 export SES_PORT=25
-export SMTP_USERNAME=AKIAIUWYA3RHAHIAP62Q
-export SMTP_PASSWORD=Apv5OSMNgaqncYUlfl2VLnIsklyULMBI9i50VNEhEWA7
+export SMTP_USERNAME=AKIAI7P76NEEEBT6QGKQ
```

Comment:
@nelsonic there is a contract in place - https://github.com/dwyl/hq/issues/329 - it appears this was a mistake. Please let me know if the keys need resetting or how I can help with this. In order to prevent this from happening in future, is there a tool you know of that can flag or block the pushing of commits with keys before it occurs? If not, I will search for one.